helm/velero-app/values.yaml

```diff
@@ -6,8 +6,7 @@
 # enabling restic). Required.
 image:
   repository: velero/velero
```

Comment:
do we want to retag the image and use quay?

Reply:
don't know - do we? 😄

Reply:
https://github.com/giantswarm/retagger/pull/539